modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/DataStreamsSnapshotsIT.java

```diff
@@ -256,7 +256,8 @@ public void testSnapshotAndRestoreInPlace() {
         RolloverRequest rolloverRequest = new RolloverRequest("ds", null);
         RolloverResponse rolloverResponse = client.admin().indices().rolloverIndex(rolloverRequest).actionGet();
         assertThat(rolloverResponse.isRolledOver(), is(true));
-        assertThat(rolloverResponse.getNewIndex(), equalTo(DataStream.getDefaultBackingIndexName("ds", 2)));
+        String backingIndexAfterSnapshot = DataStream.getDefaultBackingIndexName("ds", 2);
+        assertThat(backingIndexAfterSnapshot, equalTo(backingIndexAfterSnapshot));
```

Comment:
Was this intentional? Isn't this just testing that String.equals() works?

Reply:
Oops, you are right, let me fix this.